docs/concepts/spsp.md

```diff
@@ -0,0 +1,21 @@
+## SPSP
+
+[SPSP](https://github.com/interledgerjs/ilp-protocol-spsp) (Simple Payment Setup Protocol) is an 
+[application layer protocol](https://github.com/interledger/rfcs/blob/master/0001-interledger-architecture/0001-interledger-architecture.md#protocol-layers), 
+which uses HTTPS endpoints to securely exchange payment details, and then allows the user to send and receive value 
+over Interledger. 
+
+SPSP provides lightweight client/server APIs to send and receive value. An SPSP client sends (pay) value, and an 
```

Comment:
'SPSP provides lightweight client/server APIs to send and receive value. In a "push payment" scenario, the SPSP client sends money to the SPSP server using Interledger. In a "pull payment" scenario, the SPSP server sends money to the SPSP client using Interledger. Details in the SPSP API let the server describe which mode it supports.'